src/libcore/iter/traits/iterator.rs

```diff
@@ -2119,6 +2119,7 @@ pub trait Iterator {
     ///
     /// `find()` is short-circuiting; in other words, it will stop processing
     /// as soon as the closure returns `true`.
+    /// `iter.find(f)` is equivalent to `iter.filter(f).next()`.
```

Comment:
Can this go in its own paragraph near the end? It's not really related to the short-circuiting.